server/resources/UserResources.py

```diff
@@ -0,0 +1,60 @@
+from flask_restful import Resource, reqparse
+from models.UserModel import UserModel
+from flask_jwt_extended import (create_access_token, create_refresh_token, jwt_required, jwt_refresh_token_required, get_jwt_identity, get_raw_jwt)
+
+parser = reqparse.RequestParser()
+parser.add_argument('email', help = 'This field cannot be blank', required = True)
+parser.add_argument('password', help = 'This field cannot be blank', required = True)
+parser.add_argument('fName')
+parser.add_argument('lName')
+parser.add_argument('confirm_pass')
+
+class UserRegister(Resource):
+    def post(self):
+        data = parser.parse_args()
+
+        if UserModel.find_by_email(data['email']):
+            return {'message': 'Email {} already exists'. format(data['email'])}, 400
+
+        if len(data['password']) < 6:
+             return {'message': 'Password should be at least 6 characters long'}, 400
+
+        if data['password'] != data['confirm_pass']:
+            return {'message': 'Passwords do not match'}, 400
+
+        new_user = UserModel(
+            email = data['email'],
+            fName = data['fName'],
+            lName = data['lName'],
+            password = UserModel.generate_hash(data['password'])
+        )
+        try:
+            new_user.save_to_db()
+            access_token = create_access_token(identity = data['email'])
+            refresh_token = create_refresh_token(identity = data['email'])
```

Comment:
I feel like we should have one token that expires in like 30 days, and we should id as the identity